src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -1930,6 +1930,12 @@ namespace Microsoft.FSharp.Collections
         /// <param name="endIndex">The end index.</param>
         /// <returns>The sub list specified by the input indices.</returns>
         member GetSlice : startIndex:int option * endIndex:int option -> 'T list  
+
+        /// <summary>Get the index corresponding to ^i.</summary>
```

Comment:
The summary comment should not mention the syntax used. It should just state the intent of the extension method in the context of the type it's extending.